client/state/sites/actions.js

```diff
@@ -89,7 +89,7 @@ export function requestSites() {
 				include_domain_only: true,
 				site_activity: 'active',
 				fields:
-					'ID,URL,name,capabilities,jetpack,visible,is_private,is_vip,icon,plan,jetpack_modules,single_user_site,is_multisite,options', //eslint-disable-line max-len
+					'ID,URL,name,capabilities,jetpack,visible,is_private,is_vip,icon,plan,jetpack_modules,single_user_site,is_multisite,options,lang,post_count', //eslint-disable-line max-len
```

Comment:
We should also update `client/state/jetpack-connect/actions` to keep the field in sync.

Reply:
Good catch! c7797762315e9972b8b02e2c9a4099bc2c22b8d7

Reply:
You might be interested with a patch I'm doing in #23684, where I move these fields definitions to a `constants.js` file and import it both in `state/jetpack-connect` and in `state/sites`.

Also, the list is now in a diff-friendly form:
```js
exports const SITE_REQUEST_FIELDS = [
  'ID',
  'URL',
  ...
].join();
```

Reply:
@jsnajdr oh, much nicer! Happy to leave this and rebase once #23684 lands 👍

Reply:
How close is #23684 ? @jsnajdr If you don't mind, let's 🚢 this one and update 23684.

Reply:
@gwwar @kwight #23684 is ready for review -- if anyone from Lannister wants to have a look, I'd appreciate it. I have many more Post Editor PRs in the pipeline (the Flux removal project) and it's a very complex and mission-critical component...

Don't worry much about coordinating -- I'll resolve the conflicts after this one is merged. Just being aware that conflicts (and winter) are coming helps me a lot.